app/src/ui/preferences/accessibility.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react'
+import { DialogContent } from '../dialog'
+import { Checkbox, CheckboxValue } from '../lib/checkbox'
+
+interface IAccessibilityPreferencesProps {
+  readonly underlineLinks: boolean
+  readonly onUnderlineLinksChanged: (value: boolean) => void
+}
+
+export class Accessibility extends React.Component<
+  IAccessibilityPreferencesProps,
+  {}
+> {
+  public constructor(props: IAccessibilityPreferencesProps) {
+    super(props)
+  }
+
+  public render() {
+    return (
+      <DialogContent>
+        <div className="advanced-section">
+          <h2>Accessibility</h2>
+          <Checkbox
+            label="Underline links"
+            value={
+              this.props.underlineLinks ? CheckboxValue.On : CheckboxValue.Off
+            }
+            onChange={this.onUnderlineLinksChanged}
+          />
+          <p className="git-settings-description">
```

Comment:
```suggestion
          <Checkbox
            label="Underline links"
            value={
              this.props.underlineLinks ? CheckboxValue.On : CheckboxValue.Off
            }
            onChange={this.onUnderlineLinksChanged}
            ariaDescribedBy="underline-setting-description"
          />
          <p id="underline-setting-description" className="git-settings-description">
```

^ Adding that id may cause the need for linters to be rerun.